requirements.txt

```diff
@@ -1,7 +1,14 @@
-jinja2
+jinja2~=2.11.2
```

Comment:
- discussion open with Mihai as well, about the versions of the dependency libraries
- my take: requirements file shall be without version constraints (mostly) and we shall have a .lock file that injects the versions 
Mihai proposes to use poetry system for the version control (which is "too complicated because I'm lazy to learn it")

Reply:
https://python-poetry.org/docs/basic-usage/